cmd/kops/create_cluster.go

```diff
@@ -330,7 +330,7 @@ func NewCmdCreateCluster(f *util.Factory, out io.Writer) *cobra.Command {
 
 	cmd.Flags().StringVar(&options.Image, "image", options.Image, "Image to use for all instances.")
 
-	cmd.Flags().StringVar(&options.Networking, "networking", options.Networking, "Networking mode to use.  kubenet (default), classic, external, kopeio-vxlan (or kopeio), weave, flannel-vxlan (or flannel), flannel-udp, calico, canal, kube-router, romana, amazon-vpc-routed-eni, cilium, cni.")
```

Comment:
is there a reason `kopeio-vxlan (or kopeio), ` was removed? I _think_ that is [Justin's CNI](https://github.com/kopeio/networking) and if it still works I think we should keep it in the CLI docs, and if it doesn't we should discuss next steps but either way I think that would be outside the scope of this PR.

thoughts?

Reply:
kopeio works. But there is no documentation of that CNI in the kops docs, so it isn't very helpful to have users consider that option. I could even see us even narrow the list to only listing stable CNIs following #9140 (that would be the next steps, in my optinion).

If it is a blocker for this PR, I will add it back in though.

Reply:
ah, I suppose that makes sense. That sounds good to me now, thanks!